<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>842</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_min">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-image: url(:/images/min_first.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{
background-image: url(:/images/min_after.png);
background-repeat:no-repeat;
background-position:center center;


}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_max">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-image: url(:/images/max-min_first.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	background-image: url(:/images/max-min_after.png);
background-repeat:no-repeat;
background-position:center center;


}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>16</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background-image: url(:/images/close_first.png);
background-repeat:no-repeat;
background-position:center center;
border:none;

}

QPushButton:hover{

	
	background-image: url(:/images/close_after.png);
background-repeat:no-repeat;
background-position:center center;


}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,20,1">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true">QGroupBox
{
	border:none;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="toolButton_newStream">
          <property name="minimumSize">
           <size>
            <width>68</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>68</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton
{
	background-image:url(:/images/newStream_first.png);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QToolButton:hover
{
	background-image:url(:/images/newStream_second.png);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_email">
          <property name="minimumSize">
           <size>
            <width>68</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>68</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border-image: url(:/images/EmailImage/Email.png);}
QPushButton:hover{border-image: url(:/images/EmailImage/Email_hover.png);}

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_leaveMsg">
          <property name="minimumSize">
           <size>
            <width>68</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>68</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{border-image: url(:/images/leaveMsgPics/leaveMsg.png);}

QPushButton:hover
{
border-image: url(:/images/leaveMsgPics/leaveMsg-hover.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,10,3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="maximumSize">
          <size>
           <width>199</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox
{
	border:none;
}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>30</number>
            </property>
            <property name="rightMargin">
             <number>15</number>
            </property>
            <item>
             <widget class="QToolButton" name="toolBtn_jb">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{
	
	background-image: url(:/images/加班1.jpg);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QToolButton:hover
{
	
	background-image: url(:/images/加班2.jpg);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QToolButton:checked
{
	
	background-image: url(:/images/加班3.jpg);
	background-repeat:no-repeat;
    background-position:center center;
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolBtn_qj">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{
	
	
	background-image: url(:/images/请假1.jpg);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QToolButton:hover
{
	
	
	background-image: url(:/images/请假2.jpg);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QToolButton:checked
{
	
	
	background-image: url(:/images/请假3.jpg);
	background-repeat:no-repeat;
    background-position:center center;
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolBtn_wdk">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{
	
	
	background-image: url(:/images/忘打卡1.jpg);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QToolButton:hover
{
	
	
	background-image: url(:/images/忘打卡2.jpg);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QToolButton:checked
{
	
	
	background-image: url(:/images/忘打卡3.jpg);
	background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_chuChai">
              <property name="minimumSize">
               <size>
                <width>175</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	
	
	background-image: url(:/images/出差1.png);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QPushButton:hover
{
	
	
	background-image: url(:/images/出差2.png);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QPushButton:checked
{
	
	
	background-image: url(:/images/出差3.png);
	background-repeat:no-repeat;
    background-position:center center;
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_waiChu">
              <property name="minimumSize">
               <size>
                <width>175</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	
	
	background-image: url(:/images/外出1.png);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QPushButton:hover
{
	
	
	background-image: url(:/images/外出2.png);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QPushButton:checked
{
	
	
	background-image: url(:/images/外出3.png);
	background-repeat:no-repeat;
    background-position:center center;
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolBtn_cg">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>35</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{
	
	background-image: url(:/images/采购1.png);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QToolButton:hover
{
	
	
	background-image: url(:/images/采购2.png);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}
QToolButton:checked
{	
	background-image: url(:/images/采购2.png);
    background-repeat:no-repeat;
    background-position:center center;
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">selection-background-color: rgb(221, 245, 255);</string>
         </property>
         <property name="title">
          <string>电子流显示区</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(221, 245, 255);</string>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="maximumSize">
          <size>
           <width>199</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox
{
	border:none;
}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>47</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="MyToolBtn" name="toolBtn_dcl">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>75</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{	
	background-image: url(:/images/待处理.jpg);
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyToolBtn" name="toolBtn_kqhz">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>75</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{	
	
	background-image: url(:/images/考勤汇总.png);
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyToolBtn" name="toolBtn_cgjh">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>75</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{	
	
	background-image: url(:/images/BOM.png);
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyToolBtn" name="toolBtn_spkc">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>75</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{	
	
	background-image: url(:/images/商品库存.png);
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyToolBtn" name="toolBtn_ygsz">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>75</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{	
	
	background-image: url(:/images/员工设置.png);
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyToolBtn" name="toolBtn_tybh">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>75</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{	
	
	background-image: url(:/images/图样编号.png);
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyToolBtn" name="toolButton_daka">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>75</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{	
	
	background-image: url(:/images/上班打卡记录.png);
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="MyToolBtn" name="toolButton_weekReport">
              <property name="minimumSize">
               <size>
                <width>65</width>
                <height>75</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{	
	
	background-image: url(:/images/个人周报.png);
	background-repeat:no-repeat;
	background-position:center center;
	border:none;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>47</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="styleSheet">
        <string notr="true">QGroupBox
{
	border:none;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>180</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_userName">
          <property name="text">
           <string>吴胜兵</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_quitAutoRun">
          <property name="styleSheet">
           <string notr="true">QPushButton#pushButton_quitAutoRun
{
	color:rgb(38 , 133 , 227);
	background-color:transparent;
}

QPushButton#pushButton_quitAutoRun:hover
{
	color:rgb(97 , 179 , 246);
}

QPushButton#pushButton_quitAutoRun:pressed
{
	color:rgb(0 , 109 , 176);
}</string>
          </property>
          <property name="text">
           <string>退出自动登录</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>174</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_help">
          <property name="styleSheet">
           <string notr="true">QPushButton#pushButton_help
{
	color:rgb(38 , 133 , 227);
	background-color:transparent;
}

QPushButton#pushButton_help:hover
{
	color:rgb(97 , 179 , 246);
}

QPushButton#pushButton_help:pressed
{
	color:rgb(0 , 109 , 176);
}</string>
          </property>
          <property name="text">
           <string>使用手册</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>173</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_haveFun">
          <property name="styleSheet">
           <string notr="true">QPushButton#pushButton_haveFun
{
	color:rgb(38 , 133 , 227);
	background-color:transparent;
}

QPushButton#pushButton_haveFun:hover
{
	color:rgb(97 , 179 , 246);
}

QPushButton#pushButton_haveFun:pressed
{
	color:rgb(0 , 109 , 176);
}</string>
          </property>
          <property name="text">
           <string>Have Fun</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyToolBtn</class>
   <extends>QToolButton</extends>
   <header location="global">mytoolbtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
